jdbc/src/main/java/com/interface21/jdbc/core/JdbcTemplate.java

```diff
@@ -83,6 +82,19 @@ public int update(String sql, PreparedStatementSetter pss) {
         }
     }
 
+    public int update(String sql, PreparedStatementSetter pss) {
+        try (Connection conn = dataSource.getConnection()
+        ) {
+            return update(conn, sql, pss);
```

Comment:
```suggestion
        try (Connection conn = dataSource.getConnection()) {
            return update(conn, sql, pss);
```
개행이 약간 어색하게 느껴지네요! 한 줄에 써도 충분할 것 같습니다.

Reply:
오우 그러네요 바로 반영하겠습니다잇!